src/components/dishItems/components/menu/components/menuItem/MenuItem.js

```diff
@@ -0,0 +1,61 @@
+// prop-types
+import PropTypes from "prop-types";
+
+// reader
+import menuItemReader from "../../readers/MenuItemReader";
+
+// components
+import PrimaryButton from "../../../../../../commonComponents/primaryButton";
+
+// helpers
+import { getCurrencySymbol } from "../../../../../../utility/getCurrencySymbol";
+import { getDishIcon } from "../../../../../../utility/getDishIcon";
+
+// css
+import "./menuItem.css";
+
+const MenuItem = function (props) {
+  const { item } = props;
+
+  const name = menuItemReader.name(item);
+  const type = menuItemReader.type(item);
+  const currency = menuItemReader.currency(item);
+  const price = menuItemReader.price(item);
+  const description = menuItemReader.description(item);
+
+  return (
+    <div className="menu-item">
+      <div>
+        <img src={getDishIcon(type)} alt={type} className="icon" />
+        <h3>{name}</h3>
+        <p>
+          {getCurrencySymbol(currency)} {price}
```

Comment:
create a method called getFormattedCurrency which will return you currency in format along with ,